tests/kafkatest/services/security/minikdc.py

```diff
@@ -62,7 +63,7 @@ def start_node(self, node):
         self.logger.info(props_file)
 
         kafka_principals = ' '.join(['kafka/' + kafka_node.account.hostname for kafka_node in self.kafka_nodes])
-        principals = 'client ' + kafka_principals
+        principals = 'client ' + kafka_principals + self.extra_principals
```

Comment:
Good point! I'm currently passing in a string that starts with space, but thats unsafe.

Since this was committed, I'll fix in separate PR.